compiler/ballerina-backend-jvm/src/main/ballerina/src/compiler_backend_jvm/main.bal

```diff
@@ -19,6 +19,7 @@ import ballerina/bir;
 import ballerina/jvm;
 import ballerina/reflect;
 import ballerina/system;
+import ballerina/file;
```

Comment:
```suggestion
```

Reply:
This is because the file module is used in the file jvm_package_gen.bal